config/locales/en.yml

```diff
@@ -3011,7 +3011,8 @@ See the %{link} to find out more about %{sitename}'s features and to start using
           invalid: "invalid"
         quantity_adjusted: "Insufficient stock available. Line item updated to maximum available quantity."
         quantity_unchanged: "Quantity unchanged from previous amount."
-        cannot_remove_last_item: "Cannot remove last item from order. Please cancel order instead."
+        cancel_the_order: "This will cancel the current order.<br />Are you sure you want to proceed?"
```

Comment:
There's a convention to name the translation key `cancel_the_order_html` if it contains HTML. Rails then doesn't escape special characters when using `I18n.t` in Rails. It looks like the JS method doesn't do that here and it's not necessary here. But still nice to follow the convention in case we want to support that at some stage.

Reply:
Changed, thanks! 🙏